### PR TITLE
Add the FAQ to the repository home page.

### PR DESCRIPTION
**Pull request by Arthusppp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ and UI improvements), breathing new life into one of the most addictive turn-bas
     <img src="docs/images/screenshots/screenshot_battle.png?raw=true" width="512">
 </p>
 
+## FAQ
+You can find answers to the most common questions on our [**F.A.Q. page**](https://github.com/ihhub/fheroes2/wiki/F.A.Q.).
+
+
```
Comment (oleg-derevenetz) [inline README.md:19]: This extra empty line is not needed.

**Review by oleg-derevenetz — CHANGES_REQUESTED** [review]
LGTM in general, except minor formatting issue. I suppose we should also make the same changes in the docs/README.md as well?

File: README.md (hunk 1 above)
Comment (Arthusppp) [reply README.md:19]: Done.

**Review by oleg-derevenetz — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -34,7 +38,9 @@ Initially, the project was developed on [**sourceforge**](https://sourceforge.ne
 
 ## Development
```
Comment (ihhub) [inline README.md:39]: I think we should rename it into `Contribution and Development`.

File: docs/README.md [hunk 3]
```diff
@@ -34,7 +38,9 @@ Initially, the project was developed on [**sourceforge**](https://sourceforge.ne
 
 ## Development
```
Comment (ihhub) [inline docs/README.md:39]: I think we should rename it into `Contribution and Development`.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Arthusppp , I left one identical comment here. Could you please take a look?

**Comment by ihhub** [comment]
@Arthusppp , thank you so much for updating the documentation!

**Pull request merged**